package.json

```diff
@@ -1,5 +1,63 @@
 {
   "name": "horizon",
   "private": true,
-  "dependencies": {}
+  "scripts": {
+    "lint": "eslint ./app --ext ts,tsx,js,jsx --fix",
+    "prettier": "prettier --write 'app/**/*.(ts|tsx|js|jsx)'",
```

Comment:
I think you'd want an additional script like this:

https://github.com/artsy/convection/blob/master/package.json#L6

This ensures that the rake task will fail if there are prettier violations - otherwise the rake task will write those files and happily return 0!!

Reply:
Will follow up to add this & rubocop